Syndication/AzureStack.MarketplaceSyndication.psm1

```diff
@@ -560,6 +556,7 @@ function Import-AzSOfflineMarketplaceItem
     $ctx = Get-AzureRmContext
     $AccessToken = Resolve-AccessToken -Context $ctx -AccessToken $AccessToken
     $headers = @{ 'authorization' = "Bearer $AccessToken"}
+    $ArmEndpoint = $ctx.Environment.ResourceManagerUrl
```

Comment:
$ctx and $headers starts with lower-case letter, $AccessToken and $ArmEndpoint starts with upper case letter.. Can you please be consistent in coding style?